keras_nlp/models/albert/albert_classifier.py

```diff
@@ -47,11 +47,14 @@ class AlbertClassifier(Task):
     Args:
         backbone: A `keras_nlp.models.AlertBackbone` instance.
         num_classes: int. Number of classes to predict.
-        dropout: float. The dropout probability value, applied after the dense
-            layer.
         preprocessor: A `keras_nlp.models.AlbertPreprocessor` or `None`. If
             `None`, this model will not apply preprocessing, and inputs should
             be preprocessed before calling the model.
+        activation: Optional `str` or callable, defaults to `None`. The
+            activation function to use on the model outputs. Set
+            `activation=`"softmax"`` to return output probabilities.
```

Comment:
nit: I think we can delete the extra `` - `activation="softmax"`

Reply:
oops, thanks!